roxie/ccd/ccdstate.cpp

```diff
@@ -1130,6 +1130,17 @@ class CRoxieQuerySetManager : public CInterface, implements IRoxieQuerySetManage
         }
     }
 
+    virtual void preloadOnce() const
+    {
+        HashIterator elems(queries);
```

Comment:
Do we need to hold some CS for this ?  (or perhaps we already are ?)

Reply:
I think it works as a "set everything up then swap in the new QuerySetManager atomically" model - so effectively this is happening during construction.